code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -52,15 +52,21 @@
 	qdel(src)
 
 /obj/item/melee/touch_attack/mansus_fist/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	mansus_attack(target, user, proximity_flag, click_parameters, FALSE)
 
+/obj/item/melee/touch_attack/mansus_fist/afterattack_secondary(atom/target, mob/user, proximity_flag, click_parameters, secondary)
+	mansus_attack(target, user, proximity_flag, click_parameters, TRUE)
+	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+/obj/item/melee/touch_attack/mansus_fist/proc/mansus_attack(atom/target, mob/user, proximity_flag, click_parameters, secondary)
```

Comment:
Instead of a `secondary` bool flag, split the top and bottom code blocks into their own procs.

Your first after attack would look like:

```dm
foo()
bar()
```

...and the other would be


```dm
foo()
// code that only runs for secondary attacks here
bar()
```

Reply:
The problem here would be the `use_charge` variable, but you can just handle that in the afterattack and afterattack_secondary.

One important thing here is right-click should *not* perform the same as left-click, which appears to be the case now.